server/sonar-process/src/test/java/org/sonar/process/cluster/hz/TimedOutDistributedCall.java

```diff
@@ -17,14 +17,20 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import React from 'react';
+package org.sonar.process.cluster.hz;
 
-export default class ItemBoolean extends React.PureComponent {
-  render() {
-    if (this.props.value) {
-      return <i className="icon-check" />;
-    } else {
-      return <i className="icon-delete" />;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class TimedOutDistributedCall implements DistributedCall<Long> {
+  static final AtomicLong COUNTER = new AtomicLong();
+
+  @Override
+  public Long call() throws Exception {
+    long value = COUNTER.getAndIncrement();
+    if (value == 1L) {
+      // only the second call times out
+      Thread.sleep(30_000L);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2925)